### DISCUSSION TITLE
Moving my hugginface dataset under my organization

### DISCUSSION THREAD

**Discussion by Fatima-Haouari** [huggingface/datasets#4758] [Q&A]
Hi,
I added previously a dataset under hugginface/datasets [ArCOV-19](https://huggingface.co/datasets/ar_cov19). Is it possible to move my dataset under my organization [bigIR](https://huggingface.co/bigIR). Kindly advise on how to achieve this.

**Comment by mariosasko** [comment]
Hi! Moving this dataset abruptly would be a breaking change, so I suggest you create an identical repo under your org and submit a PR that deprecates the canonical implementation (we've done a similar thing for Multilingual Librispeech in https://github.com/huggingface/datasets/pull/4060, so you can borrow the code from there)

**Comment by albertvillanova** [comment]
Hi @Fatima-Haouari, we can now transfer the dataset under your organization and make a redirection from the original dataset URL, so that there is no disruption.

**Comment by albertvillanova** [comment]
I am sending an email to all the organization members, so that you can confirm if you agree with the transfer.
